### PR TITLE
Cleanup net48 code

### PR DESCRIPTION
**Pull request by pinzart90**
Cleanup the net48 preprocessor directives in the source code

### CODE REVIEW

File: src/DynamoSandbox/DynamoSandbox.csproj [hunk 1]
```diff
@@ -21,18 +21,6 @@
...
-  <ItemGroup Condition=" '$(TargetFramework)' == 'net48' ">
-    <Reference Include="PresentationCore" />
-    <Reference Include="PresentationFramework" />
-    <Reference Include="System.Configuration" />
-    <Reference Include="System.Xaml">
-      <RequiredTargetFramework>4.0</RequiredTargetFramework>
-    </Reference>
-    <Reference Include="WindowsBase" />
-    <Reference Include="Microsoft.Practices.Prism">
-        <HintPath>..\..\extern\prism\Microsoft.Practices.Prism.dll</HintPath>
```
[the remark this answers is not in the document]
Comment (pinzart90) [reply src/DynamoSandbox/DynamoSandbox.csproj:33]: I think so. Will do

**Comment by mjkkirschner** [comment]
good stuff.

File: test/DynamoCoreTests/Crypto/DynamoCyptoTests.cs
Comment (aparajit-pratap) [inline test/DynamoCoreTests/Crypto/DynamoCyptoTests.cs:1]: Don't we want to wait until we remove DynamoCrypto?

File: test/Libraries/DynamoMSOfficeTests/ExcelTests.cs [hunk 2]
```diff
@@ -9,1362 +9,11 @@
 namespace Dynamo.Tests
 {
 
-#if NET6_0_OR_GREATER
     [TestFixture, Apartment(ApartmentState.MTA)]
     public class ExcelTests
     {
     }
```
Comment (aparajit-pratap) [inline test/Libraries/DynamoMSOfficeTests/ExcelTests.cs:15]: Can you remind me why we didn't have any excel tests for net6?

Comment (pinzart90) [reply test/Libraries/DynamoMSOfficeTests/ExcelTests.cs:15]: https://github.com/DynamoDS/Dynamo/pull/14041#discussion_r1212472854
Looks like we simply forgot to do it after all projects were migrated
I will enable them

**Pull request merged**